connext_cmake_module/cmake/Modules/FindConnext.cmake

```diff
@@ -310,8 +310,10 @@ else()
 endif()
 
 if(Connext_FOUND)
-  if(NOT WIN32)
-    list(APPEND Connext_LIBRARIES "pthread" "dl")
+  if(ANDROID)
+      list(APPEND Connext_LIBRARIES "dl")
+  elseif(NOT WIN32)
+      list(APPEND Connext_LIBRARIES "pthread" "dl")
```

Comment:
Please avoid the repetition of logic (adding "dl" in two different locations). Instead change the conditions to perform to separate `APPEND`:

```
if(NOT WIN32)
  # append dl
  if(NOT ANDROID)
    # append pthread
  endif()
endif()
```

Reply:
@jayhou Friendly ping.

Reply:
@dirk-thomas I'll apply the advise later today.